bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/obscomp/SPProgram.java

```diff
@@ -1005,8 +1005,8 @@ public ParamSet getParamSet(PioFactory factory) {
 
             // Write the awarded time (for GSA only -- will be ignored on
             // import)
-            Pio.addParam(factory, paramSet, AWARDED_TIME_PROP,
-                    String.valueOf(_timeAllocation.getTotalTime()), "hours");
+            final double hrs = _timeAllocation.getSum().getTotalHours();
+            Pio.addParam(factory, paramSet, AWARDED_TIME_PROP, String.valueOf(hrs), "hours");
```

Comment:
Sorry, super minor but can we change `String.valueOf` to `Double.toString`? This has a very Java 1 feel to it.